client/documents/list/test/index.tsx

```diff
@@ -6,7 +6,7 @@
 import * as React from 'react';
 import { createEvent, fireEvent, render, screen } from '@testing-library/react';
```

Comment:
> 'createEvent' is defined but never used
> 'fireEvent' is defined but never used

Reply:
Good catch! Removed in https://github.com/Automattic/woocommerce-payments/pull/4180/commits/b78577bd2bb14ffd9247e38df5ff07f51ee57ef8